cypress/integration/utils/devices.ts

```diff
@@ -72,6 +81,24 @@ export const desktopDevicesForAppLayout = (
   return [macbook15(pageName, t, loggedIn), macbook11(pageName, t, loggedIn)];
 };
 
+export const allDevicesForAppLayout = (
+  pageName: string,
+  desktopTests: Tests,
+  mobileTests: Tests,
+  skipNavbar = false,
+  loggedIn = false,
+) => {
+  const desktopT = getAppLayoutTests(desktopTests, skipNavbar, loggedIn);
+  const iPadT = getAppLayoutTestsMobile(mobileTests, skipNavbar, loggedIn);
+  const iPhoneT = getAppLayoutTestsMobile(mobileTests, skipNavbar, loggedIn);
+  return [
+    macbook15(pageName, desktopT, loggedIn),
+    macbook11(pageName, desktopT, loggedIn),
+    iPad2(pageName, iPadT, loggedIn),
+    iPhoneX(pageName, iPhoneT, loggedIn),
+  ];
```

Comment:
Can simplify this a bit using the functions you already have:
```suggestion
  return [
    ...desktopDevicesForAppLayout(desktopTests, skipNavbar, loggedIn)
    ...mobileDevicesForAppLayout(mobileTests, skipNavbar, loggedIn)
  ];
```